app/Filament/Exports/SettingExporter.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Filament\Exports;
+
+use App\Models\Setting;
+use Filament\Actions\Exports\ExportColumn;
+use Filament\Actions\Exports\Exporter;
+use Filament\Actions\Exports\Models\Export;
+
+class SettingExporter extends Exporter
```

Comment:
Remove importer/exporter

Reply:
So you don't want an inport/export feature for the settings?